Sources/HaCWebsiteLib/Models/Workshop+init.swift

```diff
@@ -125,11 +124,11 @@ private struct WorkshopBuilder {
   }
 
   func fileServingUrl(relativePath: String) throws -> URL {
-    return try repoUrl(origin: "https://rawgit.com/", relativePath: "\(commitSha)\(relativePath)")
+    return try repoUrl(origin: "https://cdn.jsdelivr.net/gh/", relativePath: "@\(commitSha)\(relativePath)")
```

Comment:
What's the @ sign for?
(Forgive my ignorance)

Reply:
Literally the "@" symbol - it's just the required syntax that `jsDelivr` requires

Reply:
Ah, that makes sense. I thought it might have been a Swift-specific feature.
String interpolation actually seems pretty cool.